packages/atomizer/src/rules.js

```diff
@@ -667,6 +667,127 @@ module.exports = [
     },
     /**
     ==================================================================
+    BACKDROP-FILTER FUNCTIONS
+    https://drafts.fxtf.org/filter-effects-2/#BackdropFilterProperty
+    ==================================================================
+    */
+    // filter for custom
+    {
+        type: 'pattern',
+        name: 'Backdrop Filter',
+        matcher: 'Bkdp',
+        allowParamToValue: false,
+        styles: {
+            'backdrop-filter': '$0',
+        },
+        arguments: [
+            {
+                n: 'none',
+            },
+        ],
+    },
+    // blur
+    {
+        type: 'pattern',
+        name: 'Backdrop Blur (filter)',
+        matcher: 'BkdpBlur',
+        allowParamToValue: true,
+        styles: {
+            'backdrop-filter': 'blur($0)',
+        },
+    },
+    // brightness
+    {
+        type: 'pattern',
+        name: 'Backdrop Brightness (filter)',
+        matcher: 'BkdpBrightness',
+        allowParamToValue: true,
+        styles: {
+            'backdrop-filter': 'brightness($0)',
+        },
+    },
+    // contrast
+    {
+        type: 'pattern',
+        name: 'Backdrop Contrast (filter)',
+        matcher: 'BkdpContrast',
+        allowParamToValue: true,
+        styles: {
+            'backdrop-filter': 'contrast($0)',
+        },
+    },
+    // dropshadow
+    {
+        type: 'pattern',
+        name: 'Backdrop Drop shadow (filter)',
+        matcher: 'BkdpDropshadow',
```

Comment:
these are looking v long, should we follow 

`Bxsh(<custom-param>) box-shadow:`

though the filters are also long so I guess we match them...

Reply:
![Uploading Screen Shot 2022-12-05 at 4.56.20 PM.png…]()